### ISSUE TITLE
adapt-contrib-textinput

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2802]
### Subject of the issue
Missing aria-label when no prefix is defined.

**Comment by simondate** [comment]
@oliverfoster can you bump the component version too please?

**Comment by oliverfoster** [comment]
Don't we do that on release?

**Comment by moloko** [comment]
@simondate @oliverfoster yes, just in case any other PRs get raised at the same time it can get a bit confusing; easier just to sort it prior to release on the whole
